tests/suites/test_suite_platform.function

```diff
@@ -84,7 +84,16 @@ void time_delay_seconds(int delay_secs)
     sleep_ms(delay_secs * 1000);
```

Comment:
Do we really need these tests?

Historically, we had somewhat similar tests in the `timing` module. [They often failed on the CI](https://github.com/Mbed-TLS/mbedtls/issues/1517) and so we ended up removing them. I fear that we've reintroduced the problem, and this pull request is just one of many.

Reply:
Got it. 

Should I remove these tests here? Or another PR?

Reply:
Delay test were removed. 

I think we should keep `*get`

Reply:
Completely removing the tests may be too much. How about keeping the calls to the functions (so at least we know they don't e.g. crash), but not testing delays?

Possibly even check that `t1 = ms_time(); sleep(small); t2 = ms_time(); ASSERT(t2 > t1)`. @yuhaoth Can you clarify

> Built-in mbedtls_time function returns the number of seconds since the
> Epoch. That is affected by discontinuous jumps and cause test fail.
> Workaround it with 1 seconds tollerance.
> Epoch. That is affected by discontinuous jumps. And `nanosleep` use
> CLOCK_MONOTONIC(monotonically-increasing time source), That will cause
>  negative elapsed time difference.

What can cause a _negative_ elapsed time difference? E.g. Can this happen from automatic drift adjustment? I would have expected that `t2 - t1` might be less than `small`, but not negative.

Reply:
I just update the comments. But I did not mention `automatic drift adjustment`. I think that's enough.